packages/vaadin-date-picker/theme/lumo/vaadin-month-calendar-styles.js

```diff
@@ -65,6 +65,10 @@ registerStyles(
       transition: color 0.1s;
     }
 
+    [part='date']:not(:empty) {
```

Comment:
Relying on `:empty` instead of `[role='button']`. Not sure if either is really robust, or which one should be preferred.

Reply:
It should be ok to use `:empty` here as we do not have any whitespace for non-clickable dates 👍